# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/LayersSurfaces.ipdlh
# Commit: 290917deea87
# Full Hash: 290917deea879f965f6f1ad441df945c41c01c14
# Author: Jamie Nicol <jnicol@mozilla.com>
# Date: 2023-12-13 21:42:49
# Regressor Bug: 1866020
# File Overlap Count: 1
# Description:
#   Bug 1866020 - Override buggy colorspace conversion on Pixel devices. r=gw,padenot,geckoview-reviewers,owlish
#   
#   Pixel 6, 7, and 8 devices running Android 14 are affected by a bug
#   where video frames with SMPTE 432 color primaries are rendered
#   incorrectly when sampled from an external texture in GLES. To work
# ==============================================================================

diff -r 12e266e296c1 -r 290917deea87 gfx/layers/ipc/LayersSurfaces.ipdlh
--- a/gfx/layers/ipc/LayersSurfaces.ipdlh	Wed Dec 13 08:29:00 2023 +0000
+++ b/gfx/layers/ipc/LayersSurfaces.ipdlh	Wed Dec 13 09:06:29 2023 +0000
@@ -88,6 +88,7 @@
   IntSize size;
   SurfaceFormat format;
   bool continuous;
+  bool forceBT709ColorSpace;
   Matrix4x4? transformOverride;
 };
 